xds/src/main/java/io/grpc/xds/RingHashLoadBalancer.java

```diff
@@ -259,18 +296,22 @@ private void processSubchannelState(Subchannel subchannel, ConnectivityStateInfo
     if (stateInfo.getState() == TRANSIENT_FAILURE || stateInfo.getState() == IDLE) {
       helper.refreshNameResolution();
     }
-    Ref<ConnectivityStateInfo> subchannelStateRef = getSubchannelStateInfoRef(subchannel);
```

Comment:
Is there a functional change here? I can't find it.

Reply:
the only change is that:
before, we will ignore the `IDLE`  or `CONNECTING` for sticky-TF. Then finish the update.
now, we still ignore the `IDLE`  or `CONNECTING` for sticky-TF.  Then keep doing the update(to keep attempting reconnecting).

Reply:
Oh, the `return` then has a different scope. Right. Just like C. Looks good.